### PR TITLE
[10.0] New module sale_commission_brand

### PR DESCRIPTION
**Pull request by robyf70**
I've made a new commission module to calculate commission using product brand

### CODE REVIEW

File: sale_commission_brand/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+class AccountInvoiceLineAgent(models.Model):
+    _inherit = "account.invoice.line.agent"
+
+    @api.depends('invoice_line.price_subtotal')
+    def _compute_amount(self):
+        for line in self:
+            line.amount = 0.0
+            if (not line.invoice_line.product_id.commission_free and
+                line.commission) and \
+                    line.commission.commission_type == 'brand':
```
Comment (jackjack82) [inline sale_commission_brand/models/account_invoice.py:15]: @robyf70 in this way you break normal commission computation. You need to include this line within the "if commission_type == 'brand'"

**Review by jackjack82 — CHANGES_REQUESTED** [review]

File: sale_commission_brand/models/account_invoice.py (hunk 1 above)
Comment (jackjack82) [reply sale_commission_brand/models/account_invoice.py:15]: I mean line 12, "line.amount = 0.0"

Comment (robyf70) [reply sale_commission_brand/models/account_invoice.py:15]: I've just committed a new patch, please review

**Review by jackjack82 — APPROVED** [review]

**Review by giusygiallara93 — APPROVED** [review]

**Comment by jackjack82** [comment]
Please close, to be created a new PR with updated code. There is current errore on product_uom_qty of invoice line.

**Comment by pedrobaeza** [comment]
OK, closing

**Pull request closed without merging**